.github/workflows/mvnw.yml

```diff
@@ -0,0 +1,23 @@
+name: Java CI
+
+on:
+  schedule:
+    - cron: '0 16 * * *'
```

Comment:
https://crontab.guru/#0_16_*_*_* has this as every 4pm, think this might be a noon-ish deploy and not a nightly deploy. But :shrug: